canvas_modules/harness/Gruntfile.js

```diff
@@ -61,6 +61,29 @@ module.exports = function(grunt) {
 					dest: ".build"
 				}]
 			},
+			fonts: {
+				files: [{
+					expand: true,
+					flatten: false,
+					cwd: "./node_modules/@ibm/plex",
+					src: ["IBM-Plex*/**"],
+					dest: ".build/fonts"
+				},
+				{
```

Comment:
I'm a little surprised we have to flatten these. Is this what the carbon team suggests?

Reply:
Errors are fixed even after setting `flatten: false`. Updated PR with this change. 
Carbon team only suggested to use `$font-path` with  `@carbon/react` package -
```
@use "@carbon/react" as * with (
	$font-path: "/fonts"
);
```

Reply:
I'm confused why we need node_modules/@ibm/plex/IBM-Plex-Sans/fonts/split?  According to the docs we shouldn't have to modify anything when copying over.